app/components/preprint-status-banner/component.js

```diff
@@ -92,10 +74,10 @@ export default Component.extend({
         const tWorkflow = i18n.t(this.get('workflow'));
         const tStatusExplanation = i18n.t(this.get('statusExplanation'));
 
-        return `${i18n.t(this.get('baseMessage'), {name: tName, reviewsWorkflow: tWorkflow})} ${tStatusExplanation}.`;
+        return `${i18n.t(this.get('baseMessage'), { name: tName, reviewsWorkflow: tWorkflow })} ${tStatusExplanation}.`;
     }),
 
-    statusExplanation: computed('submission.provider.reviewsWorkflow', 'submission.reviewsState', function() {
+    statusExplanation: computed('submission.{provider.reviewsWorkflow,reviewsState}', function() {
```

Comment:
I didn't know about this syntax before... Should there be a space after the comma?

Reply:
Definitely looks weird, but that's correct 
> Note that the dependent keys must be together (without space) for the brace expansion to work.

https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/use-brace-expansion.md